posts/2023-08-22-23.0.0.8.adoc

```diff
@@ -0,0 +1,378 @@
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark
+// backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+With Open Liberty's new Proof Key for Code Exchange (PKCE) support in OpenID Connect clients, you can prevent authorization code interception attacks, which can occur in certain very specific scenarios. Also in this release, the `featureUtility installFeature` command has been updated but be aware that it might now download more features for you than it used to; we recommend using `featureUtility installServerFeatures` to install the specific set of features you need for a server configuration. We've also got a new guide on using OpenTelemetry and Jaeger, and, of course, we've been fixing bugs.
+
+// // // // // // // //
+// In the preceding section:
+// Leave any instances of `tag::xxxx[]` or `end:xxxx[]` as they are.
+//
+// Replace RELEASE_SUMMARY with a short paragraph that summarises the release. Start with the lead feature but also summarise what else is new in the release. You will agree which will be the lead feature with the reviewers so you can just leave a placeholder here until after the initial review.
+// // // // // // // //
+
+// // // // // // // //
+// Replace the following throughout the document:
+//   Replace 23.0.0.8 with the version number of Open Liberty, eg: 22.0.0.2
+//   Replace 23008 with the version number of Open Liberty wihtout the periods, eg: 22002
+// // // // // // // //
+
+In link:{url-about}[Open Liberty] 23.0.0.8:
+
+* <<pkce, Prevent authorization code interception attacks with PKCE support for OpenID Connect clients>>
+* <<featureinstall, Ensure sufficient features are installed when using `featureUtility installFeature` command>>
+* <<CVEs, Security Vulnerability (CVE) fixes>>
+* <<bugs, Notable bug fixes>>
+
+
+// // // // // // // //
+// If there were updates to guides since last release, keep the following, otherwise remove section.
+// // // // // // // //
+Along with the new features and functions added to the runtime, we’ve also made <<guides, updates to our guides>>.
+
+// // // // // // // //
+// In the preceding section:
+// Replace the TAG_X with a short label for the feature in lower-case, eg: mp3
+// Replace the FEATURE_1_HEADING with heading the feature section, eg: MicroProfile 3.3
+// Where the updates are grouped as sub-headings under a single heading 
+//   (eg all the features in a MicroProfile release), provide sub-entries in the list; 
+//   eg replace SUB_TAG_1 with mpr, and SUB_FEATURE_1_HEADING with 
+//   Easily determine HTTP headers on outgoing requests (MicroProfile Rest Client 1.4)
+// // // // // // // //
+
```

Comment:
No issue with leaving these in, but I do try to delete them simply to make it easier for reviewers and anyone else to navigate the post contents.
```suggestion
```